qcfractal/interface/collections/dataset.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 import pandas as pd
+import pint
```

Comment:
We cannot import pint, the 0.5 second load time is too long without lazy loading. You will need to catch this error in a different way.

Reply:
Okay, I did the following workaround:

```
            try:
                ret[column_name] *= constants.conversion_factor(
                    data.units, self.units)
            except ValueError as e:
                # This is meant to catch pint.errors.DimensionalityError without importing pint, which is too slow
                if e.__class__.__name__ == "DimensionalityError":
                    ret.rename(columns={column_name: f"{column_name} [{data.units}]"}, inplace=True)
                else:
                    raise
```

Reply:
Nasty, but ok.